hpy/devel/include/universal/hpyfunc_trampolines.h

```diff
@@ -118,4 +118,34 @@ typedef struct {
         return a.result;                                                \
     }
 
+typedef struct {
+    cpy_PyObject *arg0;
+    cpy_Py_buffer *arg1;
+    int arg2;
```

Comment:
Should we give these arguments more readable names rather than `arg0`. `arg1`? Maybe `self`, `buffer`, `flags`?

Reply:
Done, using `self`, `view`, `flags`.